internal/cli/atlas/privateendpoints/gcp/gcp.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcp
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const provider = "GCP"
+
+func Builder() *cobra.Command {
+	const use = "gcp"
+	cmd := &cobra.Command{
+		Use:   use,
```

Comment:
[style] feel free to in line it, as it is a const offer little value here 
```suggestion
	cmd := &cobra.Command{
		Use:   "gcp",
```